contracts/Surl.sol

```diff
@@ -42,4 +48,91 @@ library Surl {
         bytes memory res = vm.ffi(inputs);
         (status, data) = abi.decode(res, (uint256, bytes));
     }
+
+    function _powershell(string memory url) internal returns (uint256 status, bytes memory data) {
+        string[] memory inputs = new string[](3);
+        inputs[0] = "cmd";
+        inputs[1] = "/C";
+        inputs[2] = string.concat(
+            "cmd /C certutil -urlcache -f ",
+            url,
+            " tmpfile && type tmpfile && del tmpfile"
+        );
+        bytes memory res = vm.ffi(inputs);
+        (status, data) = abi.decode(res, (uint256, bytes));
+    }
+
+    function _cmd(string memory url) internal returns (uint256 status, bytes memory data) {
+        string[] memory inputs = new string[](3);
+        inputs[0] = "bash";
+        inputs[1] = "-c";
+        inputs[2] = string.concat(
+            "powershell -Command \"$response = Invoke-WebRequest -Uri '",
+            url,
+            "' -UseBasicParsing; $status = $response.StatusCode; $body = $response.Content -join ''; Write-Output \"$status`n$body\"\""
+        );
+        bytes memory res = vm.ffi(inputs);
+        (status, data) = abi.decode(res, (uint256, bytes));
+    }
+
+    function _isWindowsOS() internal returns (bool) {
```

Comment:
or is there an environment variable to test the kind of OS? I saw there is `OSTYPE`in unix-like, but not sure on windows though.